faq.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="./css/common/main.css">
-  <link rel="stylesheet" type="text/css" href="./css/FAQ.css">
+  <link rel="stylesheet" type="text/css" href="./css/pages/FAQ.css">
```

Comment:
Let's have filenames in lowercase also :) 
(will add this in coding style guide)